src/_includes/macros/sub-nav.html

```diff
@@ -50,10 +50,10 @@
                 </ul>
             {%- endfor %}
             </div>
-        </section>
-        <aside class="sub-nav_media">
+        </div>
+        <div class="sub-nav_media">
```

Comment:
Cool. I was worried about leaving it unlabeled with that stray h4 tag. I'll stick it back in and check it. Thanks!